lib/signature-formatter.js

```diff
@@ -0,0 +1,162 @@
+'use babel';
+
+export default class SignatureFormatter {
+  formatTypeSignatureArrows(sig) {
+    return formatTypeSignatureArrows(sig);
+  }
```

Comment:
sorry for nitpicking, but what's the use of this?

Reply:
Shoot -- So, the spec currently tests the functionality of formatTypeSignatureArrows independently since it has a lot going on.  I can rework the tests to pass through formatTypeSignature and slightly change the type signature passed in each test to just have a dummy function name. It just technically adds risk of breaking tests should the latter change instead of the former.

Reply:
Changed type sigs in relevant tests to "f :: \<existing\>" and updated expectations to align. Sorry about that.